ironfish/src/rpc/routes/accounts/getTransactions.ts

```diff
@@ -66,12 +69,41 @@ router.register<typeof GetAccountTransactionsRequestSchema, GetAccountTransactio
 
     const headSequence = await node.wallet.getAccountHeadSequence(account)
 
+    let capacity = -1
+    let queue = undefined
+    if (request.data.show) {
+      capacity = request.data.show
+      queue = new FastPriorityQueue<TransactionValue>(function (a, b) {
+        return a.transaction.expirationSequence() < b.transaction.expirationSequence()
```

Comment:
This will be a min-heap, so it will give the earliest transactions instead of the latest

I'm also wondering if `expirationSequence` is the best sort key. `TransactionValue` also has `sequence` which is the sequence of the block that contains the transaction (if any)

An `expirationSequence` of `0` also means that the transaction will never expire

Reply:
For the 1st issue, since we use `poll` func to remove element when queue is full and new element comes,  the elements remain in the queue after all transactions are handled are the latest in my opinion.
For the 2nd issue, I have tried to use `sequence` as the sort key. But there are some pending transactions who do not have sequence, but they are the latest. So `expirationSequence` is used here now.

Reply:
Updated. `sequence` as sort key for transactions already mined, `expirationSequence ` as sort key for transactions already pending and minersFee also handled.